src/Stratis.Bitcoin.Features.BlockStore/BlockRepository.cs

```diff
@@ -78,6 +78,9 @@ public interface IBlockRepository : IDisposable
         BlockStoreRepositoryPerformanceCounter PerformanceCounter { get; }
 
         bool TxIndex { get; }
+
+        /// <summary>Represents the last block stored to disk.</summary>
+        ChainedBlock HighestPersistedBlock { get; }
```

Comment:
OK please double check this, as I'm 99.9% sure that `IBlockRepository.BlockHash` IS the last block stored to disk?

Reply:
That's correct. But `BlockHash` is a hash and not a `ChainedBlock` so we can't use it

Reply:
Oh yes! I see now ... but seeing as we have the repository in PosMinting, can't we just query it from there?

Reply:
>can't we just query it from there?

Yep, we could. Good idea. 
But BlockRepository is no longer used here so not applicable.